homeassistant-supervised/DEBIAN/preinst

```diff
@@ -51,7 +51,30 @@ if [[ "$(sysctl --values kernel.dmesg_restrict)" != "0" ]]; then
     echo "kernel.dmesg_restrict=0" >> /etc/sysctl.conf
 fi
 
+# If the hassio_supervisor service is running or exists, stop it 
+if [[ "$(systemctl is-active hassio-supervisor.service)" == "active" ]]; then
+    info "Stopping hassio_supervisor service"
+    systemctl stop hassio-supervisor.service
+fi
+
+# If the hassio_apparmor service is running or exists, stop it
+if [[ "$(systemctl is-active hassio-apparmor.service)" == "active" ]]; then
+    info "Stopping hassio_apparmor service"
+    systemctl stop hassio-apparmor.service
+fi
 
+# Check for existing hassio_supervisor container and destroy it
+if [[ "$(docker ps -aq -f name=hassio_supervisor)" ]]; then
+    # ensure the hassio_supervisor service is stopped
+    info "Removing existing hassio_supervisor container"
+    docker container rm --force hassio_supervisor > /dev/null
+fi
+
+# If docker is running, stop it
+if [[ "$(systemctl is-active docker.service)" == "active" ]]; then
+    info "Stopping Docker service"
+    systemctl stop docker.service
```

Comment:
This seems a bit harsh to me, is this really required?

Reply:
You're right it's not really required, I just figured since we are restarting docker anyway after we modify the `daemon.json` we might as well stop it beforehand. However, I can see how it could be considered a bit harsh, I will remove it